This is a hacky workaround to force the build system to build the Python
bindings with the 'build' and 'installer' modules (akin to how other Python
modules are built in MassOS), instead of by running setup.py. This is because
setup.py would build the bindings as a Python EGG, which pip no longer
supports.

Note that this workaround is designed for use in the MassOS build system ONLY.
Notably, it will not support DESTDIR-based installations, the module will
refuse to (re)install itself if it already exists on the system, and it won't
support uninstalling the package using 'make uninstall'.

--- a/Makefile.am
+++ b/Makefile.am
@@ -59,11 +59,11 @@
 
 # Use distutils to build the module.
 all-local: .stamp-distutils-in-builddir config.py cupshelpers/config.py
-	$(PYTHON) setup.py build
+	$(PYTHON) -m build -nw -o dist
 
 # Use distutils to install the module.
 install-exec-local: .stamp-distutils-in-builddir
-	$(PYTHON) setup.py install --prefix=$(DESTDIR)$(prefix)
+	$(PYTHON) -m installer --compile-bytecode 1 dist/*.whl
 
 # Uninstall the module, crossing our fingers that we know enough
 # about how distutils works to do this.  Unfortunately, distutils
